extensions/ql-vscode/workspace-databases-schema.json

```diff
@@ -44,6 +44,72 @@
       },
       "required": ["repositoryLists", "owners", "repositories"],
       "additionalProperties": false
+    },
+    "local": {
+      "type": "object",
+      "properties": {
+        "lists": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string"
+              },
+              "databases": {
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "name": {
+                      "type": "string"
+                    },
+                    "dateAdded": {
+                      "type": "number"
+                    },
+                    "language": {
+                      "type": "string"
```

Comment:
We could force to only allow certain values to be set here e.g. javascript etc. 

You can do that using `enum` in JSON schema e.g. `"enum": ["javascript", "..."]`

Reply:
I was wondering about this too, but I'm currently not actually sure what languages to include 😅 E.g. if someone's working on CodeQL support for a new language, or doing something with a custom version of the CLI, we don't necessarily know what language their database is for 🤔 

I'm thinking that we'd check the language anyway (when validating against the dbscheme), so it might be fine to keep as `string`? 

(I might be overthinking this though 😆 )

Reply:
Does the extension "jus work" if someone is working on support for a new language?

Reply:
> Does the extension "just work" if someone is working on support for a new language?

Yep, language support doesn't depend on the extension (unless we've hard-coded languages somewhere), so people should be able to use custom CLI or databases without issue (at least locally)!

Reply:
Hmm okay let's leave it as a string for now but we might need to revisit.